Tests/scripts/update_id_set.py

```diff
@@ -219,7 +219,7 @@ def get_script_data(file_path, script_code=None):
 
 def get_depends_on(data_dict):
     depends_on = data_dict.get('dependson', {}).get('must', [])
-    depends_on_list = list(set([cmd.split('|')[-1] for cmd in depends_on]))
+    depends_on_list = list(set(cmd.split('|')[-1] for cmd in depends_on))
```

Comment:
Shouldn't we split on 3 |||| ?

Reply:
actually yes, did not looked at the logic of the code, just fixed the lint suggestion.
either way it works the same